homeassistant/components/filter/sensor.py

```diff
@@ -80,9 +80,7 @@
 NAME_TEMPLATE = "{} filter"
 ICON = "mdi:chart-line-variant"
 
-FILTER_SCHEMA = vol.Schema(
-    {vol.Optional(CONF_FILTER_PRECISION, default=DEFAULT_PRECISION): vol.Coerce(int)}
-)
+FILTER_SCHEMA = vol.Schema({vol.Optional(CONF_FILTER_PRECISION): vol.Coerce(int)})
```

Comment:
Just a note for a follow-up PR, since precision is now a user-facing sensor option, this can be completely dropped in a future PR.